bears/css/CSSCombBear.py

```diff
@@ -0,0 +1,224 @@
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.bears.requirements.NpmRequirement import NpmRequirement
+import json
+
+
+@linter(executable='csscomb',
+        output_format='corrected',
+        use_stdout=True,
+        use_stdin=False,
+        result_message='Change coding style based on CSS rules written in ' +
+                       '.csscomb.json',
+        )
+class CSSCombBear:
+    """
+    CSScomb is a coding style formatter for CSS.
+
+    For more information, consult <https://github.com/csscomb/csscomb.js>.
+    """
+
+    LANGUAGES = {"CSS"}
+    REQUIREMENTS = {NpmRequirement('csscomb')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    ASCIINEMA_URL = 'https://asciinema.org/a/bxpke7nizyxdf6mlx6ss4h405'
+    CAN_FIX = {'Formatting'}
+
+    @staticmethod
+    def create_arguments(filename, file, config_file):
+        return (filename,)
+
+    def generate_config(filename, file,
+                        verbose: bool = True,
+                        remove_empty_rulesets: bool = True,
+                        always_semicolon: bool = True,
+                        color_case: str = "upper",
+                        block_indent: str = "\\t",
+                        color_shorthand: bool = False,
+                        eof_newline: bool = True,
+                        leading_zero: bool = False,
+                        quotes: str = "single",
+                        sort_order_fallback: bool = True,
+                        space_before_colon: str = " ",
```

Comment:
same for these ones. i think it should be True or False.

if the program asks me: Do you want a space before colon?
you would say Yes/True, not ``" "`` :D